mlir/include/mlir/Conversion/MIOpenPasses.td

```diff
@@ -38,24 +38,6 @@ def ConvertMIOpenToGPU : Pass<"convert-miopen-to-gpu", "ModuleOp"> {
   ];
 }
 
-//===----------------------------------------------------------------------===//
-// MIOpenToGPU within an existing GPUModule
-//===----------------------------------------------------------------------===//
-
-def ConvertMIOpenWithinGPUModule : Pass<"test-miopen-lowering-gpu-module", "gpu::GPUModuleOp"> {
-  let summary = "Lower the operations from the MIOpen dialect within an "
```

Comment:
This pass doesn't have a consumer in our lowering pipeline, which is why you removed it?

Reply:
This pass was the main reason we still had a `LowerMIOpenOps.h`, isn't consumed anywhere but some old test code, and was very much not being maintained to keep up with pipeline changes

Reply:
Sounds good.